tests/test_sycl_queue.py

```diff
@@ -364,6 +366,25 @@ def test_proj(device):
     assert_sycl_queue_equal(result_queue, expected_queue)
 
 
+@pytest.mark.parametrize(
+    "device",
+    valid_devices,
+    ids=[device.filter_string for device in valid_devices],
+)
+def test_rsqrt(device):
+    X = [1.0, 8.0, 27.0]
+    Y = [1.0, 0.35355338, 0.19245009]
+
+    x = dpnp.array(X, device=device)
+    result = dpnp.rsqrt(x)
+    expected = dpnp.array(Y)
```

Comment:
what if to use numpy rather than hardcode expected values:
```suggestion
    X = [1.0, 8.0, 27.0]

    x = dpnp.array(X, device=device)
    result = dpnp.rsqrt(x)
    expected = 1 / numpy.sqrt(x.asnumpy())
```